src/packet/vp9.rs

```diff
@@ -371,6 +375,7 @@ impl Vp9Depacketizer {
         vp9_extra.pid = self.picture_id;
         vp9_extra.layers_widths.copy_from_slice(&self.width[..3]);
         vp9_extra.layers_heights.copy_from_slice(&self.height[..3]);
+        vp9_extra.is_keyframe |= !self.p && (self.sid == 0 || !self.l) && self.b;
```

Comment:
I don't understand, why this is `|=` ?

Reply:
[In this loop](https://github.com/algesten/str0m/blob/c76993cf7b0c69d6b221979bffa4a73c6f88ccab/src/packet/buffer_rx.rs#L235-L244) the `CodecExtra` can be overwritten, and we do not want `is_keyframe: true` to be replaced with `is_keyframe: false`.

Reply:
Those in the loop are all packets of the same "picture".

I think this means we are reporting this media sample as a keyframe if any of the inner spatial layers contains a keyframe, but it's unclear which one. Correct me if I am wrong. I think, either this flag should refer to spatial layer 0 alone, or maybe we should state which spatial layers contains the keyframe ?

Reply:
Ah, I got confused :) I see why this works now.